main.js

```diff
@@ -10,5 +20,53 @@ function handleFoodChange() {
 
 async function fetchRecipe(food) {
   //--- write your code below ---
+  let response = await fetch(
+    `https://api.edamam.com/search?q=${food}&app_id=${YOUR_APP_ID}&app_key=${YOUR_APP_KEY}`
+  );
+  let data = await response.json();
+  // console.log(data);
+  // console.log(data.hits[1].recipe.label);
+  // document.getElementById("recipe-label").innerHTML = data.hits[1].recipe.label;
+  // console.log(data.hits[1].recipe.url);
+  // document.getElementById("recipe-label").href = data.hits[1].recipe.url;
   //--- write your code above ---
+  endPoint.innerHTML = "";
+  getMeSomeRecipes(data);
+  console.log(data);
+}
+// next step is to put the recipe label and url in the a tah in inner.html
+// items that we want to be displayed in order:
+//
+// Recipe img -> findings.hits[i].recipe.image
+// Recipe label with href -> label findings.hits[i].recipe.label ; href -> findings.hits[i].recipe.url
+// Calories -> findings.hits[i].recipe.calories
+
+function getMeSomeRecipes(findings) {
```

Comment:
An interesting use of parameter here. Make sure that they are as clear as possible, it tend to go for 'result' but that's personal preference.